code/modules/projectiles/ammunition/magazines.dm

```diff
@@ -341,7 +341,7 @@ obj/item/ammo_box/magazine/tommygunm45
 
 /obj/item/ammo_box/magazine/m12g/breach
 	name = "shotgun magazine (12g breacher slugs)"
-	icon_state = "m12gbc"
+	icon_state = "m12bc"

```

Comment:
don't change the icon_state name, change the sprite name, every other shotgun magazine use the same nomenclature.